common/reviews/api/heft-config-file.api.md

```diff
@@ -12,13 +12,16 @@ export class ConfigurationFile<TConfigurationFile> {
     getObjectSourceFilePath<TObject extends object>(obj: TObject): string | undefined;
     getPropertyOriginalValue<TParentProperty extends object, TValue>(options: IOriginalValueOptions<TParentProperty>): TValue;
     // (undocumented)
-    loadConfigurationFileAsync(configurationFilePath: string): Promise<TConfigurationFile>;
-    }
+    loadConfigurationFileForProjectAsync(projectPath: string): Promise<TConfigurationFile>;
+    tryLoadConfigurationFileForProjectAsync(projectPath: string): Promise<TConfigurationFile | undefined>;
+}
 
 // @beta (undocumented)
 export interface IConfigurationFileOptions<TConfigurationFile> {
+    disableRigs?: boolean;
```

Comment:
The name `disableRigs` seems misleading, since this setting only affects the one config file; it's not "disabling rigs" in general.

I'm also thinking rig support should be opt-in rather than opt out. Nearly all of our config files will opt-in.  But when defining a config file, it seems better to declare what it is, rather than what it isn't.   It would also help someone who is searching the code base to find config files that use rigs, for example.

Some possible names:
- `supportsRigs: true`
- `riggable: true`